plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1014,8 +1014,10 @@ public void testPredicateModuloPushdown()
         if (!hasBehavior(SUPPORTS_PREDICATE_MODULO_PUSHDOWN)) {
             return;
         }
-
+        //modulo over bigint
         assertThat(query("SELECT orderkey FROM orders WHERE orderkey % 4 = 0")).isFullyPushedDown();
+        //modulo over int
+        assertThat(query("SELECT shippriority FROM orders WHERE shippriority % 4 = 0")).isFullyPushedDown();
```

Comment:
see also https://github.com/trinodb/trino/pull/12104 for other useful tests

Reply:
thx, added all the tests from your PR